mex/common/models/base/merged_item.py

```diff
@@ -3,3 +3,7 @@
 
 class MergedItem(BaseEntity):
     """Base model for all merged item classes."""
+
+
+class PreviewItem(BaseEntity):
```

Comment:
I am surprised to find this in `merged_item.py`. Could this go into it's own `preview_item.py` module? Or rename `merged_item.py` to `items.py`?

Reply:
done in c15a4a7 - among some further clean-up